app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,27 @@
+class StaticPagesController < ApplicationController
+  skip_before_action :require_login, :only => [:home]
+  def home
+    @user = User.new
+    redirect_to user_path(current_user.id) if signed_in_user?
+  end
+
+  def timeline
+    @profile = false
```

Comment:
Ah they do. Perhaps you could add a `before_action: set_profile_to_false` at the top.